app.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "tableau-connector",
+    "description": "data.world tableau web data connector",
+    "repository": "https://github.com/datadotworld/tableau-connector",
+    "scripts": {},
+    "env": {
```

Comment:
`REACT_APP` here implies that we'll run the app in dev mode (as opposed to the prod-optimized build). I'm not sure what the right answer is here, but one possibility would be to retire `REACT_APP_OAUTH_REDIRECT_URI` and `REACT_APP_OAUTH_ROOT_URL` in favor or relative redirects, and configure the OAuth client with multiple redirect URLs, so that `REACT_APP_OAUTH_CLIENT_ID` and `REACT_APP_OAUTH_CLIENT_SECRET` could be the same for staging and prod. That way we can build the app using a heroku pre-release task and trust that the same slug will work in staging and prod. This setup would also require a Procfile, to start the app using `serve -s build` (making sure that `serve` is a dependency listed in package.json).